sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTestBase.java

```diff
@@ -402,11 +396,11 @@ void importCertificateRunner(Consumer<ImportCertificateOptions> testRunner) {
     }
 
     @Test
-    public abstract  void importPemCertificate() throws IOException;
+    public abstract  void importPemCertificate(HttpClient httpClient, CertificateServiceVersion serviceVersion) throws IOException;
 
     void importPemCertificateRunner(Consumer<ImportCertificateOptions> testRunner) throws IOException {
 
-        byte[] certificateContent = readCertificate("pemCert.pem");
+        byte[] certificateContent = readCertificate("certificate.pem");
```

Comment:
The file name was changed. Was this intended?

Reply:
This is merge issue. Thanks for catching this.